ethcore/engines/clique/src/lib.rs

```diff
@@ -762,9 +762,8 @@ impl Engine for Clique {
 		}
 	}
 
-	fn set_signer(&self, signer: Box<dyn EngineSigner>) {
-		trace!(target: "engine", "set_signer: {}", signer.address());
```

Comment:
we might keep this log, I think we could have something like:

```rustc
		if let Some(signer) = signer.as_ref() {
			trace!(target: "engine", "set_signer: {:?}", signer.address());
		} else {
		  // maybe read self.signer.read() and print if the signer was set to Some(signer) before	
          trace!(target: "engine", "clear_signer");
		}

```

Reply:
do you mind putting it in an `if log_enabled!(target: "engine", log::Level::Trace)` block so we don't perform these operations when the log is disabled?

Thanks